pkgs/development/ocaml-modules/ipaddr/default.nix

```diff
@@ -1,20 +1,22 @@
-{ stdenv, fetchurl, buildDunePackage, sexplib, ppx_sexp_conv }:
+{ stdenv, fetchurl, buildDunePackage, ocaml
```

Comment:
The `fetchurl`, `ocaml` and `sexplib` arguments are not used.

Reply:
Yikes, sorry to force several rounds of this on you. Thanks for staying on top of things.
I just found https://github.com/Synthetica9/nix-linter and used that to find any pesky leftovers.